core/src/test/scala/com/microsoft/azure/synapse/ml/lime/LIMESuite.scala

```diff
@@ -43,7 +44,7 @@ trait LimeTestBase extends TestBase {
 }
 
 @deprecated("Please use 'com.microsoft.azure.synapse.ml.explainers.TabularLIME'.", since="1.0.0-rc3")
-class TabularLIMESuite extends EstimatorFuzzing[TabularLIME] with
+class TabularLIMESuite extends PyEstimatorFuzzing[TabularLIME] with
```

Comment:
I think the class heirarchy isnt as expected here which is why you are still generating tests for these. Take a look at serenas comment and double check the class heirarchy here